### ISSUE TITLE
Bug you can't avoid

### ISSUE THREAD

**Issue by AmanoTC** [SparrowBrain/Playnite.GGDeals#14]
This is an information to you: There will be a major problem with importing Playstation games to GG.Deals and you can do nothing to avoid it.

Problem:
If you own any of the Playstation Plus plans (in my case, Premium) and you try a game from the catalogue for a limited time (download or stream doesn't matter) or just play a game within the provided catalogue, Sony "adds" that game to your games as "owned", although you don't own it.
This is 100% reproducable, since Sony even sends you an Email with "Your transaction has been successful. You have been charged 0,00 Dollar/Euro/Yen."
So, in reality, you never bought that game and if the limited time is over or you cancel Playstation Plus, you are no longer able to play the game, since you don't really own it.

This leads to severe problems, because Sony reports such games as owned and Playnite, logically, picks up those games as owned. Thus your extension also picks up those games as owned and reports it back to GG.Deals as owned. While, in reality, you don't own it and maybe want to buy it and shop for deals. Which you will no longer see at GG.Deals.

Microsoft handles the "owning" the same way for games you try out via Gamepass, but I have to do further testing if the problem still persists if you uncheck "XCloud" within the settings of your extension.
The "Sony" Extension for Playnite doesn't differentiate between really owned games and cloud games. I guess this is because Sony itself doesn't differentiate between them and sends no divided data that can be progressed by any extension.

As I have pointed out, this is a major problem for GG.Deals and I see no way for you to work around this. You would have to contact Josef Nemec, the Playnite dev, or at least the dev of the "Sony" Playnite-Extension to work together with them. But as long as Sony doesn't change any of their practices or at least sends divided data, I see no way to make this work.

**Comment by SparrowBrain** [comment]
Yeah, not much we can do about it.

You can always remove/hide those games in Playnite, but it's not going to be practical when you have thousands of games, as well, it will not work on automatic sync.

I'll see, if there's anything I could do about it, but probably not :/

**Comment by SpazShark** [comment]
This applies to GamePass as well correct? In my limited experience it seems to be pulling everything down, both owned games and games that I have subscription-based access to from GP.

**Comment by SparrowBrain** [comment]
I suggest switching off automatic sync in addon settings. Then you can curate your Playnite library before syncing with GG.deals.

Probably gonna make automatic syncing off by default too.

**Comment by SpazShark** [comment]
Yeah I just have the console game syncs off on Playnite (Xbox, PS, Nintendo) for now as I had already manually added everything and the times that I buy things on console these days is rare.
The real value for with this tool is to scoop up all those games outside of GOG and Steam. Great job @SparrowBrain !!!

**Comment by SparrowBrain** [comment]
From version v2.1.1 the auto-sync for new games is not automatically enabled (affects new users). This is so new users can clean up their libraries before syncing with GG.deals.

**Comment by C4rl0s79** [comment]
just ad excusions to what is synced based on library and / or platform. I do not want to clean my library as adding again ton of emulated games takes lots of time. And handling emulated games by playnite is headache on its own.

**Comment by SparrowBrain** [comment]
@C4rl0s79 filtering by library is configurable via GG.deals settings. Emulated games should be exluded by default (Playnite library).
